data/changelog.txt

```diff
@@ -80,6 +88,19 @@ Overhaul Blazemod and split into seperate mods.
 Polymorph spells to transform your foes.
 Spells using components system.
 Aftershock: Add autodoc tiers and augmentation clinics.
+Adds location variants with dinosaurs.
+Aftershock: Megacity oriented overmap generation adjustments
+Aftershock: New Melee oriented bionics
+Adds Acrocanthosaurus, dino baby overhaul.
+Add New Black Dragon Mutation Line.
+Adjust fog frequency in black dragon lair, minor map adjustments.
+Adds monsters 'Scutellosaurus', 'Maiasaura', and 'Coelophysis'.
+[Magiclysm] Attunements - Special abilities gained by mastering two schools of magic.
+Add monsters 'Pachyrhinosaurus', 'Pentaceratops', and 'Torosaurus'
+Organizes dinos into three main factions with defined behavior
+Rebalance Jurassic spawns and monsters for realism, fix predator in-fighting.
+Cap monsters at 1000 kg so butchering kind of works.
+[magiclysm] implement summonable motorcycle
```

Comment:
```suggestion
[Magiclysm] implement summonable motorcycle
```
Alternatively, "Magiclysm:" as for Aftershock (or else change the Aftershock ones).